testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java

```diff
@@ -997,4 +992,39 @@ public void testNoUserProfileProviderComponentUponRealmChange() {
         // this used to return non-empty collection
         assertThat(adminClient.realm(realmName).components().query(null, UserProfileProvider.class.getName()), empty());
     }
+
+    @Test
+    public void setEmptyAttributeValues() {
+        String realmName = "new-realm";
+        RealmRepresentation rep = new RealmRepresentation();
+        rep.setRealm(realmName);
+        rep.setAttributes(new HashMap<>());
+        rep.getAttributes().put("myboolean", "");
+        rep.getAttributes().put("mylong", "");
+        rep.getAttributes().put("myint", "");
+        rep.getAttributes().put(RealmAttributes.ACTION_TOKEN_GENERATED_BY_USER_LIFESPAN + ".something", "");
+
+        adminClient.realms().create(rep);
+
+        adminClient.realm("new-realm");
```

Comment:
Nitpick, I would use realmName.

Reply:
Would have possibly changed that, if it wasn't for the fact that will result in a new CI run ;)